upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -168,22 +168,46 @@ func NewAWSCloud(region string, tags map[string]string) (AWSCloud, error) {
 
 		requestLogger := newRequestLogger(2)
 
-		c.cf = cloudformation.New(session.New(), config)
+		sess, err := session.NewSession()
+		if err != nil {
+			return &awsCloudImplementation{}, err
```

Comment:
Dumb question but why are we returning a new struct?

Reply:
No good reason, so I've changed it to use the extant "c" variable.